lib/companies/companies.ex

```diff
@@ -139,12 +142,12 @@ defmodule Companies.Companies do
     |> exclude(:left_join)
     |> distinct([c, _i, _j], true)
     |> join(:inner, [c, _i, _j], j in assoc(c, :jobs))
-    |> order_by([_c, _i, j], desc: j.inserted_at)
+    |> order_by([_c, _i, j], asc: c.name)
```

Comment:
Why did you change this? We want the newest jobs first, we don't care about the company name.

Reply:
Point taken, though some users might want an option. In any case, the sort key should be obvious to the user. So, I'd suggest that you add the submission date to the metadata. That way, the user can easily figure out if a job has already been skimmed or is simply too old to be of interest.